lib/matplotlib/projections/polar.py

```diff
@@ -235,7 +235,6 @@ def get_tick_space(self):
 class ThetaLocator(mticker.Locator):
     """
     Used to locate theta ticks.
-
```

Comment:
Please keep the empty lines. [PEP-257](https://www.python.org/dev/peps/pep-0257/):

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.